pkg/asset/internal/templates.go

```diff
@@ -156,6 +156,33 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 `)
 
+	KubeletApproverTemplate = []byte(`apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kubelet-approver
+  namespace: kube-system
+  labels:
+    app: kubelet-approver
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: kubelet-approver
+      labels:
+        app: kubelet-approver
+    spec:
+      containers:
+      - name: kubelet-approver
+        image: quay.io/coreos/kapprover:latest
```

Comment:
I could tag it. I didn't tag it just yet because I wanted to check first if there were any modification you guys would require.